src/bot.ts

```diff
@@ -172,7 +189,7 @@ export class DiscordBot {
       const channel = guild.channels.get(room);
       if (channel) {
         const lookupResult = new ChannelLookupResult();
-        lookupResult.channel = channel;
+        lookupResult.channel = channel as any;
```

Comment:
This seems off - why is it needed? If this could be unset, shouldn't the interface have then like `Discord.TextChannel | undefined` or something?